backend/httpclient/http_client_test.go

```diff
@@ -10,10 +10,34 @@ import (
 )
 
 func TestNewClient(t *testing.T) {
-	t.Run("New() without any opts should return http.DefaultClient", func(t *testing.T) {
-		client, err := New()
+	t.Run("New() without any opts should return expected http client and transport", func(t *testing.T) {
+		client, err := New(Options{
+			Timeouts: &TimeoutOptions{
+				Timeout:               time.Second,
+				DialTimeout:           7 * time.Second,
+				KeepAlive:             2 * time.Second,
+				TLSHandshakeTimeout:   3 * time.Second,
+				ExpectContinueTimeout: 4 * time.Second,
+				MaxConnsPerHost:       10,
+				MaxIdleConns:          5,
+				MaxIdleConnsPerHost:   7,
+				IdleConnTimeout:       6 * time.Second,
+			},
+			Middlewares: []Middleware{},
+		})
```

Comment:
we should call `New` with no options here to test this behavior.

Reply:
good catch! copied too much from the other test